package.json

```diff
@@ -37,12 +35,18 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^6.7.2",
+    "@babel/preset-env": "^7.8.4",
+    "@babel/preset-react": "^7.8.3",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.1",
+    "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
     "eslint-plugin-import": "^2.19.1",
+    "eslint-plugin-jest": "^23.8.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^1.7.0",
-    "prettier": "1.19.1"
+    "prettier": "1.19.1",
+    "react-test-renderer": "^16.12.0"
```

Comment:
Are you using this one `react-test-renderer`? I think `@testing-library/react` is enough.

Reply:
Removed.